packages/material-ui/src/TextField/TextField.test.js

```diff
@@ -190,7 +190,7 @@ describe('<TextField />', () => {
         </TextField>,
       );
 
-      const input = container.querySelector('input[type="hidden"]');
+      const input = container.querySelector('input[aria-hidden]');
```

Comment:
Would it be enough? Why is the nature of the input important?

```suggestion
      const input = container.querySelector('input');
```

Reply:
> Would it be enough? Why is the nature of the input important?

For validation and forms.